### PR TITLE
New PS cmdlet and updates to existing 

### PR DESCRIPTION
**Pull request by JensTrier**
- Adding Get-CsPhoneNumberAssignment and referencing it in Set/Remove-CsPhoneNumberAssignment
- Adding deprecation notice to Get-CsOnlineTelephoneNumber and a reference to Get-CsPhoneNumberAssignment
- Updating Import-CsOnlineAudioFile to talk about automatic marked for deletion of AA and CQ audio files
- Adding information about the exported audio format in Export-CsOnlineAudioFile

### CODE REVIEW

File: skype/skype-ps/skype/Import-CsOnlineAudioFile.md [hunk 1]
```diff
@@ -127,6 +127,11 @@ This cmdlet supports the common parameters: `-Debug, -ErrorAction, -ErrorVariabl
 ### Microsoft.Rtc.Management.Hosted.Online.Models.AudioFile
 
 ## NOTES
+
+When you are importing an audio file to be used for Auto Attenant or Call Queue, the audio file will automatically be marked for deletion (as seen by running
```
Comment (get-itips) [inline skype/skype-ps/skype/Import-CsOnlineAudioFile.md:131]:
```suggestion
When you are importing an audio file to be used for Auto Attendant or Call Queue, the audio file will automatically be marked for deletion (as seen by running
```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CapabilitiesContain
+Filters the returned results based on the capabilities assigned to the phone number. You can specify one or more capabilities delimited by comma. Supported capabilities
+are ConferenceAssignment, VoiceApplicationAssignment, UserAssignment and OCMobile.
+
+If you specify only one capability, you will get all phone numbers returned that have that capability assigned. If you specify a comma seperated list for instance like
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:139]:
```suggestion
If you specify only one capability, you will get all phone numbers returned that have that capability assigned. If you specify a comma separated list for instance like 

```

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9 @dariomws

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+Applicable: Microsoft Teams
+
+Required: False
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AssignedPstnTargetId
+Filters the returned results based on the user or resource account id the phone number is assigned to. Supported values are UserPrincipalName, SIP address and ObjectId.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:121]:
```suggestion
Filters the returned results based on the user or resource account ID the phone number is assigned to. Supported values are UserPrincipalName, SIP address, and ObjectId.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+Applicable: Microsoft Teams
+
+Required: False
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CapabilitiesContain
+Filters the returned results based on the capabilities assigned to the phone number. You can specify one or more capabilities delimited by comma. Supported capabilities
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:136]:
```suggestion
Filters the returned results based on the capabilities assigned to the phone number. You can specify one or more capabilities delimited by a comma. Supported capabilities

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 5]
```diff
@@ -0,0 +1,395 @@
...
+are ConferenceAssignment, VoiceApplicationAssignment, UserAssignment and OCMobile.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:137]:
```suggestion
are ConferenceAssignment, VoiceApplicationAssignment, UserAssignment, and OCMobile.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 6]
```diff
@@ -0,0 +1,395 @@
...
+Applicable: Microsoft Teams
+
+Required: False
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -NumberType
+Filters the returned results based on the number type. Supported values are DirectRouting, CallingPlan and OperatorConnect.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:202]:
```suggestion
Filters the returned results based on the number type. Supported values are DirectRouting, CallingPlan, and OperatorConnect.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 7]
```diff
@@ -0,0 +1,395 @@
...
+Applicable: Microsoft Teams
+
+Required: False
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PstnAssignmentStatus
+Filters the returned results based on the assignment status. Support values are Unassigned, UserAssigned, ConferenceAssigned, VoiceApplicationAssigned and ThirdPartyAppAssigned
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:217]:
```suggestion
Filters the returned results based on the assignment status. Support values are Unassigned, UserAssigned, ConferenceAssigned, VoiceApplicationAssigned, and ThirdPartyAppAssigned.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 8]
```diff
@@ -0,0 +1,395 @@
...
+Applicable: Microsoft Teams
+
+Required: False
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Skip
+Skipping the first X returned results and the default value is 0.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:232]:
```suggestion
Skips the first X returned results and the default value is 0.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 9]
```diff
@@ -0,0 +1,395 @@
...
+
+### None
+
+## OUTPUTS
+
+### ActivationState
+The activation state of the telephone number.
+
+### AssignedPstnTargetId
+The Id of the object the phone number is assigned to.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:351]:
```suggestion
The ID of the object the phone number is assigned to.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 10]
```diff
@@ -0,0 +1,395 @@
...
+
+### Capability
+The list of capabilities assigned to the phone number.
+
+### CivicAddressId
+The Id of the CivicAddress assigned to the phone number.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:357]:
```suggestion
The ID of the CivicAddress assigned to the phone number.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 11]
```diff
@@ -0,0 +1,395 @@
...
+
+### IsoCountryCode
+The ISO country code assigned to the phone number.
+
+### LocationId
+The Id of the Location assigned to the phone number.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:363]:
```suggestion
The ID of the Location assigned to the phone number.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 12]
```diff
@@ -0,0 +1,395 @@
...
+The type of the phone number.
+
+### PortInOrderStatus
+The status of any port in order covering the phone number.
+
+### PstnAssignmentStatus
+The assignment status of the phone number.
+
+### PstnPartnerId
+The Id of the PSTN partner providing the phone number.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:378]:
```suggestion
The ID of the PSTN partner providing the phone number.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 13]
```diff
@@ -0,0 +1,395 @@
...
+
+### PstnPartnerName
+The name of the PSTN partner
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:381]:
```suggestion
The name of the PSTN partner.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 14]
```diff
@@ -0,0 +1,395 @@
...
+
+### TelephoneNumber
+The phone number. The number is always displayed with prefixed "+", even if it was not assigned using prefixed "+".
+
+The object returned is of type SkypeTelephoneNumberMgmtCmdletAcquiredTelephoneNumber.
+
+## NOTES
+The cmdlet is available in Teams PowerShell module 4.0.0 or later.
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:390]:
```suggestion
```

File: skype/skype-ps/skype/Import-CsOnlineAudioFile.md [hunk 15]
```diff
@@ -62,7 +62,7 @@ This example uses the `-AsByteStream` parameter instead of the `-Encoding byte`
 ## PARAMETERS
 
 ### -ApplicationId
-The ApplicationId parameter is the identifier for the application which will use this audio file. For example, if the audio file will be used with an organizational auto attendant, then it needs to be set to "OrgAutoAttendant". If the audio file will be used with a hunt group (call queue), then it needs to be set to "HuntGroup". If the audio file will be used with Microsoft Teams, then it needs to be set to "TenantGlobal"
+The ApplicationId parameter is the identifier for the application which will use this audio file. For example, if the audio file will be used with an Auto Attendant, then it needs to be set to "OrgAutoAttendant". If the audio file will be used with a Call Queue, then it needs to be set to "HuntGroup". If the audio file will be used with Microsoft Teams, then it needs to be set to "TenantGlobal"
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Import-CsOnlineAudioFile.md:65]:
```suggestion
The ApplicationId parameter is the identifier for the application which will use this audio file. For example, if the audio file will be used with an Auto Attendant, then it needs to be set to "OrgAutoAttendant". If the audio file will be used with a Call Queue, then it needs to be set to "HuntGroup". If the audio file will be used with Microsoft Teams, then it needs to be set to "TenantGlobal."
```

File: skype/skype-ps/skype/Import-CsOnlineAudioFile.md [hunk 16]
```diff
@@ -127,6 +127,11 @@ This cmdlet supports the common parameters: `-Debug, -ErrorAction, -ErrorVariabl
 ### Microsoft.Rtc.Management.Hosted.Online.Models.AudioFile
 
 ## NOTES
+
+When you are importing an audio file to be used for Auto Attendant or Call Queue, the audio file will automatically be marked for deletion (as seen by running
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Import-CsOnlineAudioFile.md:131]:
```suggestion
When you import an audio file to be used for Auto Attendant or Call Queue, the audio file will automatically be marked for deletion (as seen by running
```

File: skype/skype-ps/skype/export-csonlineaudiofile.md [hunk 17]
```diff
@@ -84,6 +84,9 @@ Accept wildcard characters: False
 ### Byte[]
 
 ## NOTES
+The audio content generated by Export-CsOnlineAudioFile is always in WAV format (PCM 16 bit and mono) irrespective on which format the audio was imported as.
+Therefore please ensure that the file extension used to store the content is WAV.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/export-csonlineaudiofile.md:88]:
```suggestion
Therefore, ensure that the file extension used to store the content is WAV.
```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 18]
```diff
@@ -0,0 +1,395 @@
...
+ms.author: jenstr
+ms.reviewer: 
+manager:
+schema: 2.0.0
+---
+
+# Get-CsPhoneNumberAssignment
+
+## SYNOPSIS
+This cmdlet will display information about one or more phone numbers.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:16]:
```suggestion
This cmdlet displays information about one or more phone numbers.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 19]
```diff
@@ -0,0 +1,395 @@
...
+### Example 6
+```powershell
+Get-CsPhoneNumberAssignment -ActivationState Activated -CapabilitiesContain VoiceApplicationAssignment -PstnAssignmentStatus Unassigned
+```
+This example returns information about all activated phone numbers with the capability VoiceApplicationAssignment that are not assigned.
+
+## PARAMETERS
+
+### -ActivationState
+Filters the returned results based on the number type. Supported values are Activated, AssignmentPending, AssignmentFailed, UpdatePending and UpdateFailed.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:106]:
```suggestion
Filters the returned results based on the number type. Supported values are Activated, AssignmentPending, AssignmentFailed, UpdatePending, and UpdateFailed.

```

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 20]
```diff
@@ -0,0 +1,395 @@
...
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CapabilitiesContain
+Filters the returned results based on the capabilities assigned to the phone number. You can specify one or more capabilities delimited by comma. Supported capabilities
+are ConferenceAssignment, VoiceApplicationAssignment, UserAssignment and OCMobile.
+
+If you specify only one capability, you will get all phone numbers returned that have that capability assigned. If you specify a comma separated list for instance like 
+
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:140]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

Thanks @JohanFreelancer9! Done.

**Review by dariomws — APPROVED** [review]

**Comment by JensTrier** [comment]
@yogkumgit please do not merge this PR until further notice by me or Michael Chan

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by JensTrier** [comment]
@tiburd @yogkumgit NO  - please do NOT merge this until you hear from me or Michael Chan

**Comment by JensTrier** [comment]
@tiburd please merge this PR

**Pull request merged**